lib/OrdinaryDiffEqSSPRK/Project.toml

```diff
@@ -15,6 +15,7 @@ RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Static = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
```

Comment:
This should be moved to the `[extras]` section below and be included in the `test = [...]` target.

Reply:
Thanks for the catch!